### ISSUE TITLE
Poor results with MS-SSIM optimization

### ISSUE THREAD

**Issue by trougnouf** [liujiaheng/compression#7]
Do you have any advice regarding MS-SSIM optimization? I get consistently poor results when optimizing directly for the MS-SSIM. Do you have to train longer, mix with other losses, or switch to MS-SSIM later in training in order to get the results described?

**Comment by liujiaheng** [comment]
For the ms-ssim experiments, you should change the distortion loss to (1-msssim) and fine-tune the pretrained model optimized by MSE loss of lambda 8192 for 50w iters with learning rate of 1e-5. Then, you should and the lambda should be divide 128 or 64 according to your requirements. 

You can find this training strategy in our EDIC paper. The training strategy is similar with ICLR 18.

**Comment by trougnouf** [comment]
Thank you !
